### PR TITLE
Add "only" option

### PR DESCRIPTION
**Pull request by jannismilz**
Hey @robertogallea, just came across this package and wanted to implement it. (Great idea, I love it!)

I really wanted the option "only" instead of "ignore" so I implemented it.

Would love a review and if I forgot anything 🙌
Feel free to ask any questions.

### CODE REVIEW

**Comment by robertogallea** [comment]
Hi @jannismilz !
I think it is quite a useful addition, thank you! ~~I'll merge it right now and create a new version as soon as possible.~~

Thanks again!

Please address the comments ;)

File: README.md [hunk 1]
```diff
@@ -26,9 +26,13 @@ Add the provided recorder to your `config/pulse.php` configuration file as follo
             'enabled' => env('PULSE_DB_SIZE_RECORDER_ENABLED', true), // recorder switch
             'throttle' => env('PULSE_DB_SIZE_RECORDER_THROTTLE', 15), // throttling time before refreshing table size
             'connection' => env('PULSE_DB_SIZE_RECORDER_CONNECTION'), // database connection to use
+            'only' => [
+                // prioritized if not empty
+                // only select these tables
+            ]
```
Comment (robertogallea) [inline README.md]: I am from mobile but it looks like there is a missing comma.

**Review by robertogallea — CHANGES_REQUESTED** [review]

File: src/Recorders/DBSizeRecorder.php [hunk 2]
```diff
@@ -31,12 +31,30 @@ public function record(IsolatedBeat $event)
...
+                ->map(function (string $table) {
+                    return "'$table'";
+                })
+                ->implode(',');
+
+            $operator = null;
+
+            // "only" is prioritized because of like "least-selection" principle
+            if (!empty($onlyTables)) {
+                $operator = 'IN'
```
Comment (robertogallea) [inline src/Recorders/DBSizeRecorder.php]: Missing semicolon

File: src/Recorders/DBSizeRecorder.php [hunk 3]
```diff
@@ -31,12 +31,30 @@ public function record(IsolatedBeat $event)
...
+            } else if (!empty($ignoredTables)) {
+                $operator = 'NOT IN'
```
Comment (robertogallea) [inline src/Recorders/DBSizeRecorder.php]: Missing semicolon

**Review by robertogallea — CHANGES_REQUESTED** [review]

**Review by jannismilz — COMMENTED** [review]
Thanks for the rapid response! 

Sorry my bad, I fixed the typos.

**Comment by robertogallea** [comment]
Unfortunately tests do not pass yet.

**Comment by jannismilz** [comment]
> Unfortunately tests do not pass yet. 

Gonna fix that tomorrow. Was expected..

**Comment by jannismilz** [comment]
@robertogallea I think I found the problem. Should be fixed now.

**Comment by jannismilz** [comment]
@robertogallea any update?

**Review by robertogallea — APPROVED** [review]

**Comment by robertogallea** [comment]
Well done! Thanks!

**Comment by robertogallea** [comment]
> @robertogallea any update?

Merged and v1.1.0 published

**Pull request merged**